tsconfig.json

```diff
@@ -24,7 +24,7 @@
     "baseUrl": ".",
     "moduleResolution": "node",
     "resolveJsonModule": true,
-    "types": ["node", "mocha"],
+    "types": ["node", "mocha", "@types/meteor-mdg-validated-method"],
```

Comment:
TypeScript seems unable to find this automatically because it's in `node_modules/@types/meteor-mdg-validated-method` instead of, like, `node_modules/@types/meteor/mdg:validated-method` or something.